common/src/node.rs

```diff
@@ -98,7 +151,47 @@ impl Graph {
             .expect("coords are in-bounds");
 
         *voxel = block_update.new_material;
+        *modified = true;
         *old_surface = surface.take().or(*old_surface);
+
+        self.remove_adjacent_margins(block_update.chunk_id);
+        true
+    }
+
+    /// Removes margins from any populated adjacent chunks. When a chunk is modified, this function should
+    /// be called on that chunk to ensure that adjacent chunks are invisible under the potentially false assumption
+    /// that they are hidden by world generation.
```

Comment:
This comment is indeed confusing. I think I missed a "not" somewhere, but adding it will make the sentence structure have too many negatives in it. I'll need to reword it.

The bug this would prevent is as follows: If the player digs down and reaches the edge of a solid chunk, if the case is not handled properly, that solid chunk will remain solid and will therefore not be visible, and you'll see sky instead.